activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Require `belongs_to` association by default.
+    This can by configure by `config.active_record.require_belongs_to` option
+    or by `optional: true` option on relation.
+
+    *Josef Šimánek*
```

Comment:
Actually the final note is not true right now. `config.active_record.require_belongs_to = true` is set directly in railtie, not in application config. Should I move it to new generated `application.rb` and let it `nil` for existing applications? In that case your note will be true.